### DISCUSSION TITLE
How do you get webp imports to work in unit tests?

### DISCUSSION THREAD

**Discussion by kaiyoma** [vitest-dev/vitest#3442] [Q&A]
I am migrating to both Vite and Vitest (at the same time) and my UI has components that import `.webp` files. The UI and Vitest are sharing the same config and the UI is working fine, but some of my tests are failing on the imports:

```
TypeError: Unknown file extension ".webp" for C:\Users\kgetz\Work\event-viewer\lib\geiger-vendor\dist\images\arista\5310-F.webp
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: {
  "code": "ERR_UNKNOWN_FILE_EXTENSION",
}
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

 Test Files  1 failed (1)
```

What's the correct way to fix this?

**Comment by AriPerkkio** [comment]
@kaiyoma does this work out-of-the-box with latest Vitest version, `v0.31.3`? There were some changes how files under `**/dist/**` are treated.

**Reply by kaiyoma** [reply]
Yes, this works now. I had opened an issue about this (https://github.com/vitest-dev/vitest/issues/3444) and one of the developers pushed a fix.
